src/core/reducers/versions.js

```diff
@@ -40,6 +40,7 @@ export type AddonFileType = {|
   is_mozilla_signed_extension: boolean,
   is_restart_required: boolean,
   is_webextension: boolean,
+  optional_permissions: Array<string>,
```

Comment:
`permissions` is optional apparently. Is `optional_permissions` really available all the time?

Reply:
Yes, I just wrote the code for it, so I know that it will always be available.